src/MOIWrapper.jl

```diff
@@ -8,7 +8,15 @@ const CbcCI = CbcCInterface
 
 mutable struct Optimizer <: MOI.AbstractOptimizer
     inner::CbcModel
-    Optimizer() = new(CbcModel()) # Initializes with an empty model
+    Optimizer(::Nothing) = new(CbcModel()) # Initializes with an empty model
```

Comment:
Why is the ``::Nothing`` needed?

Reply:
To differentiate the default constructor to the one with kwargs.. I took CPLEX.jl example. Apparently I cannot define a default constructor that takes kwargs.

Reply:
But then tests fail for julia 0.6

Reply:
Could you remove the inner constructor?

Then it should be valid to call `Optimizer(CbcModel())` from the outer constructor that takes keyword arguments.

Reply:
Yes, you are right. I did it